### ISSUE TITLE
Tor connectivity when running as VM

### ISSUE THREAD

**Issue by adevoss** [raspiblitz/raspiblitz#1821]
Fresh install on vm and running build_sdcard.sh
Read similar issue #1816 and purged and reinstalled tor (0.4.4.6-1~d10.buster+1)
```
dial tcp 127.0.0.1:9050: connect: connection refused
err='Tor tcp connection refused'
```
```
netstat -tulpn
tcp        0      0 127.0.0.1:9050          0.0.0.0:*               LISTEN      21460/tor
```

**Comment by adevoss** [comment]
```
curl --socks5 localhost:9050 --socks5-hostname localhost:9050 -s https://check.torproject.org/ | cat | grep -m 1 Congratulations | xargs
Congratulations. This browser is configured to use Tor.
```

**Comment by rootzoll** [comment]
v1.6.2 is out that should already be better fitted for VM use.

And for v1.7 there will be a big refactor of the setup process ... will make this issue a reminder to recheck the VM build and its Tor connectivity.

**Comment by openoms** [comment]
possibly related: https://github.com/rootzoll/raspiblitz/issues/2531

**Comment by openoms** [comment]
Closing as should be fixed by afbc484. Continue in #2531 if the issue persists.
